app/validators/fee/interim_fee_validator.rb

```diff
@@ -8,9 +8,11 @@ def self.fields
   end
 
   def validate_quantity
-    if @record.is_disbursement? || @record.is_interim_warrant?
+    if qauntity_must_be_zero?
```

Comment:
**[typo]** _quantity_ instead of _qauntity_

Reply:
ta